cmd/api/src/api/v2/anonymize.go

```
@@ -0,0 +1,312 @@
+	common.DisplayName,
+	common.Description,
+	common.Email,
+	common.OperatingSystem,
+	common.Title,
+}
+
+// generatePseudonym creates a random hex pseudonym with the given prefix.
+func generatePseudonym(prefix string) string {
+	b := make([]byte, 4)
+	if _, err := rand.Read(b); err != nil {
+		slog.Warn("generatePseudonym: crypto/rand.Read failed, using fallback", slog.String("prefix", prefix), attr.Error(err))
```

golangci-lint (2.11.4)

[major] message should be capitalized

(sloglint)